deepchem/models/dft/nnxc.py

```diff
@@ -0,0 +1,125 @@
+from abc import abstractproperty, abstractmethod
+import torch
+import numpy as np
+from typing import Union, Iterator, List
+from dqc.xc.base_xc import BaseXC
+from dqc.utils.datastruct import ValGrad, SpinParam
+from dqc.utils.safeops import safenorm, safepow
+from dqc.api.getxc import get_xc
+
+
+class NNLDA(BaseXC, torch.nn.Module):
+    """Neural network xc functional for LDA
+    neural network xc functional of LDA (only receives the density as input)
+    Parameters
+    ----------
+    nnmodel: torch.nn.Module
+        Neural network for xc functional
+    ninpmode: int
+        The mode to decide the transformation of the density to NN input.
+    outmultmode: int
+        The mode to decide Eks from NN output
+    """
+
+    def __init__(self,
+                 nnmodel: torch.nn.Module,
+                 ninpmode: int = 1,
+                 outmultmode: int = 1):
+        super().__init__()
+        self.nnmodel = nnmodel
+        self.ninpmode = ninpmode
+        self.outmultmode = outmultmode
+
+    @property
+    def family(self) -> int:
+        return 1
+
+    def get_edensityxc(
```

Comment:
Should this method be public or private? Do we expect a user to use this?

Reply:
No, but it is used in HybridXC .